src/main/java/seedu/address/model/meeting/meeting/Location.java

```diff
@@ -11,6 +11,7 @@
  */
 public class Location {
     public static final String EMPTY_LOCATION_FIELD = "-";
+    // TODO: remove after allowing blank optional fields to pass.
     public static final String MESSAGE_CONSTRAINTS = "Locations can take any values, and it should not be blank";
```

Comment:
Missing period

Reply:
Message will be removed later on, so will not be making extra changes.